packages/server/lib/modes/run.js

```diff
@@ -1652,14 +1651,20 @@ module.exports = {
   },
 
   async run (options) {
-    // electron >= 5.0.0 will exit the app if all browserwindows are closed,
-    // this is obviously undesirable in run mode
-    // https://github.com/cypress-io/cypress/pull/4720#issuecomment-514316695
-    app.on('window-all-closed', () => {
-      debug('all BrowserWindows closed, not exiting')
-    })
+    try {
+      const { app } = require('electron')
+
+      // electron >= 5.0.0 will exit the app if all browserwindows are closed,
+      // this is obviously undesirable in run mode
+      // https://github.com/cypress-io/cypress/pull/4720#issuecomment-514316695
+      app.on('window-all-closed', () => {
+        debug('all BrowserWindows closed, not exiting')
+      })
 
-    await app.whenReady()
+      await app.whenReady()
+    } catch {
```

Comment:
i think there should be some filtering on the err here, surely we don't want to catch every possible error in this block here

Reply:
or this try/catch can be narrowed to just line 1655

Reply:
Done